pilot/pkg/features/pilot.go

```diff
@@ -200,6 +200,12 @@ var (
 			"if headless services have a large number of pods.",
 	).Get()
 
+	EnableRemoteJwks = env.RegisterBoolVar(
+		"PILOT_JWT_ENABLE_REMOTE_JWKS",
+		false,
+		"If enabled, treats the Issuer in RequestAuthentication as a mesh cluster and configures Remote Jwks in Envoy",
```

Comment:
the description needs a bit update, the issuer is always a url (not the istio cluster name) and the flag just switches to let envoy to fetch the jwks instead of istiod.

Reply:
Updated